src/includes/configuration/before-send-fingerprint/android.mdx

```diff
@@ -0,0 +1,14 @@
+```java
+import io.sentry.android.core.SentryAndroid;
+import android.database.sqlite.SQLiteException;
+import java.util.Arrays;
+
+SentryAndroid.init(options -> {
```

Comment:
Thanks this reminded me that we still are missing this on Apple: https://github.com/getsentry/sentry-docs/pull/2462